examples/demo_all.pp

```diff
@@ -47,5 +47,8 @@
   include ciscopuppet::demo_vpc_domain
   include ciscopuppet::demo_vrf
   include ciscopuppet::demo_vtp
-  include ciscopuppet::demo_vxlan
+  # Conditionally include ciscopuppet::demo_vxlan
+  if platform_get() =~ /n(3|9)k/ {
```

Comment:
With the commit of https://github.com/cisco/cisco-network-puppet-module/pull/172, this will have to be refactored a bit.

1) Move this call into `demo_all_cisco.pp`
2) Change call to `include ciscopuppet::cisco::demo_vxlan`